persistence/README.md

```diff
@@ -145,6 +145,12 @@ For example, on macOS, you can check for this with `lsof -i:5432` and kill the a
 **Q**: Why not use an ORM?
 **A**: We are trying to keep the module small and lean initially but are ope
 
+**Q**: What is a `Param` in the `Gov` schema?
+**A**: It represents a value associated with a name and a height that we can reference to represent governance settings. These settings have the power of altering the behaviour of various aspects of the network.
+
+**Q**: What is a `Flag` in the `Gov` schema?
+**A**: A flag is very much alike a `Param` with the difference that it also has a boolean flag to specify if that setting is enabled or not at any point in time (height). We are discussing if we should replace the boolean flag and allow multivariate feature flags.
+
```

Comment:
@andrewnguyen22: I have added some color here about flags/params.

Everything (very little 😅) I know about flags comes from this comment https://github.com/pokt-network/pocket/issues/76#issue-1213446455 and subsequent conversations with @Olshansk in the form of comments to this PR. 

One thing I assumed initially, that I later changed to the current code, was that flags shouldn't have a value like params but just a boolean flag. Maybe the truth is a bit of both: we should remove the `enabled` in flags and create logic around the `value` that we can define as float to handle multivariate flags later on, perhaps this would be out of scope for this PR but I am happy to change things if needed.

As things stand, these descriptions should fit, even though we are not yet using flags anywhere so I am assuming that this is very much subject to change as we iterate regardless.

Reply:
Thanks for the extra context. I think the flag concept will definitely evolve but the current description and implementation is more than sufficient to move forward.